.github/workflows/release.yml

```diff
@@ -84,20 +86,24 @@ jobs:
               generate_release_notes: true
             })
 
-      - name: Check commit messages for feat or break
+      - name: Check commit messages and descriptions for feat or break
         id: check_commits
         run: |
```

Comment:
It's recommended to include the commit description along with the commit message for better context. You can use the COMMIT_DESCRIPTION variable for this purpose.